classes/log/td.php

```diff
@@ -4,9 +4,9 @@
 
 namespace Fluentd\Log;
 
-class Td extends \Fuel\Core\Log{
+class Td extends \Fuel\Core\Log {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition